src/less/textbox/base/textbox.less

```diff
@@ -79,7 +79,7 @@ textarea.textbox__control {
     }
 
     &:focus {
-        .border-color-token(textbox-focus-border-color, color-action-primary);
+        .border-color-token(textbox-focus-border-color, color-black);
```

Comment:
It seems we have replaced a product level token with a primitive here. Ideally the two params should be (component-level, product-level). Would `@color-form-control-border` work? If in doubt we could also wait and do it as part of https://github.com/eBay/skin/issues/1646

Reply:
So there is no token for focused border (border non focused is different from focused one)
So I added `@color-form-control-focus-border` token